cloud/openstack/os_server.py

```diff
@@ -18,6 +18,7 @@
 # You should have received a copy of the GNU General Public License
 # along with this software.  If not, see <http://www.gnu.org/licenses/>.
 
+import six
```

Comment:
you are adding a new requirement here, the cloud modules normally execute on the master but not always, you cannot assume six is installed.